
/usr/local/lib/python3.10/dist-packages/pytorch_lightning/loggers/wandb.py:389: There is a wandb run already in progress and newly created instances of `WandbLogger` will reuse this run. If this is not desired, call `wandb.finish()` before instantiating `WandbLogger`.
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
┏━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┳━━━━━━━━┓
┃[1m    [22m┃[1m Name                        [22m┃[1m Type             [22m┃[1m Params [22m┃
┡━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━╇━━━━━━━━┩
│ 0  │ bert                        │ BertModel        │ 89.1 M │
│ 1  │ classifiers                 │ ModuleList       │ 12.6 M │
│ 2  │ hidden_layer1               │ ModuleList       │  4.2 M │
│ 3  │ hidden_layer2               │ ModuleList       │  4.1 K │
│ 4  │ sigmoid                     │ Sigmoid          │      0 │
│ 5  │ criterion                   │ BCELoss          │      0 │
│ 6  │ metrics                     │ MetricCollection │      0 │
│ 7  │ metrics_per_label_accuracy  │ MetricCollection │      0 │
│ 8  │ metrics_per_label_precision │ MetricCollection │      0 │
│ 9  │ metrics_per_label_recall    │ MetricCollection │      0 │
│ 10 │ metrics_per_label_f1score   │ MetricCollection │      0 │
└────┴─────────────────────────────┴──────────────────┴────────┘
[1mTrainable params[22m: 23.9 M
[1mNon-trainable params[22m: 82.0 M
[1mTotal params[22m: 105 M
[1mTotal estimated model params size (MB)[22m: 423
/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/call.py:54: Detected
KeyboardInterrupt, attempting graceful shutdown...
[37mSanity Checking[39m [35m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[39m [37m2/2[39m [37m0:00:01 • 0:00:00[39m [37m38.39it/s
[?25h
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
Error executing job with overrides: []
Traceback (most recent call last):
  File "/content/drive/MyDrive/murata_labo_exp/murata_labo_exp_src/exp4/main.py", line 606, in main
    trainer.test(model, data_module)
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/trainer.py", line 754, in test
    return call._call_and_handle_interrupt(
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/call.py", line 44, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/trainer.py", line 794, in _test_impl
    results = self._run(model, ckpt_path=ckpt_path)
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/trainer.py", line 989, in _run
    results = self._run_stage()
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/trainer.py", line 1028, in _run_stage
    return self._evaluation_loop.run()
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/loops/utilities.py", line 182, in _decorator
    return loop_run(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/loops/evaluation_loop.py", line 134, in run
    self._evaluation_step(batch, batch_idx, dataloader_idx, dataloader_iter)
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/loops/evaluation_loop.py", line 405, in _evaluation_step
    call._call_callback_hooks(trainer, hook_name, output, *hook_kwargs.values())
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/call.py", line 208, in _call_callback_hooks
    fn(trainer, trainer.lightning_module, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/callbacks/progress/rich_progress.py", line 561, in on_test_batch_end
    assert self.test_progress_bar_id is not None
AssertionError
Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
[?25h